src/binder/binder_context.cpp

```diff
@@ -50,7 +51,7 @@ bool BinderContext::GetColumnPosTuple(
     type::TypeId& value_type) {
   auto db_id = std::get<0>(table_id_tuple);
   auto table_id = std::get<1>(table_id_tuple);
-  auto schema = catalog::Catalog::GetInstance()
+  auto schema = catalog::CatalogStorageManager::GetInstance()
```

Comment:
There is a `StorageManager` under storage that handles very low-level storage stuff, such as cpu instructions for various storage media. I'm not sure if this code is actually used, but it is there.